tests/test_misc.py

```diff
@@ -16,11 +16,15 @@ def test_version():
     assert len(sver) > 0
     assert iver > 0
 
+# global variable for error_cb call back function
+seen_error_cb = False
+
 def test_error_cb():
     """ Tests error_cb. """
 
     def error_cb(error_msg):
-        print('OK: error_cb() called')
+        global seen_error_cb
+        seen_error_cb = True
         assert error_msg.code() in (confluent_kafka.KafkaError._TRANSPORT, confluent_kafka.KafkaError._ALL_BROKERS_DOWN)
 
     conf = {'bootstrap.servers': 'localhost:9093', # Purposely cause connection refused error
```

Comment:
This is actually probably the second most likely port to find a broker on localhost, so maybe you want to connect to localhost:22 or somesuch that will definately not a Kafka broker.